docs/pages/identity-security/session-summaries.mdx

```diff
@@ -0,0 +1,339 @@
+---
+title: Session Recording Summaries
+description: Describes how to generate and view session recording summaries using a language model.
```

Comment:
This description makes it sound like we're going to walk the user through a set of steps to generate summaries, but that mostly happens automatically.


What if we said this instead?


```suggestion
description: Describes how to use Teleport Identity security to summarize session recordings with a language model.
```